src/Model/Translatable/TranslationInterface.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the KnpDoctrineBehaviors package.
+ *
+ * (c) KnpLabs <http://knplabs.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Knp\DoctrineBehaviors\Model\Translatable;
+
+/**
+ * Translation interface.
+ *
+ * Should be used to tag translation entities.
+ */
+interface TranslationInterface
+{
+    /**
+     * Returns the translatable entity class name.
+     *
+     * @return string
+     */
+    public static function getTranslatableEntityClass();
+
+    /**
+     * Returns object id.
+     *
+     * @return mixed
+     */
+    public function getId();
+
+    /**
+     * Sets entity, that this translation should be mapped to.
+     *
+     * @param TranslatableInterface $translatable The translatable
+     *
+     * @return $this
+     */
+    public function setTranslatable($translatable);
```

Comment:
Can you typehint the parameter ?

Reply:
I added it but I'm not sure if I should add other missing types (notably return types) in this PR or create a dedicated one, mostly because I would have to modify traits (I wouldn't make the diff cluttered). WDYT?

Reply:
Thanks, yes I agree that return typehints should be added in a separate PR :)